frontend/cypress/e2e/FormBCSCPage.cy.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable no-undef */
 describe("BCSC Form", () => {
   beforeEach(() => {
+    cy.intercept("GET", "/api/clients/districts?page=0&size=250", {
```

Comment:
We changed the URL so it's now `/api/districts` instead of `/api/clients/districts`